mat-process-api.Tests/V1/Controllers/ProcessDataImagesControllerTests.cs

```diff
@@ -175,5 +182,329 @@ public void given_an_exception_thrown_check_that_the_controller_throws_correct_e
             Assert.AreEqual(expectedStatusCode, result.StatusCode);
         }
         #endregion
+
+        #region Get Process Image
+
+        [Test]
+        public void given_a_valid_request_when_GetProcessImage_controller_method_is_called_then_it_returns_a_200_Ok_response()
+        {
+            //arrange
+            var request = MatProcessDataHelper.CreateGetProcessImageRequestObject();
```

Comment:
This is a nitpicky thing, but it helps a ton, when reading tests. I've seen there were a lot of changes from:
`var request = new GetProcessImageRequest();`, to:
`var request = MatProcessDataHelper.CreateGetProcessImageRequestObject();`

The reason why this line: `var request = new GetProcessImageRequest();` was left as is rather than changed to be generated from helper was that it helps to signify that this test does not depend on what the request object is, and that the only reason the request object is there is to keep the compiler happy so that the test could be run.

Now that it's changed the helper, it's no longer clear if the contents of the request object are of any significance to the test. Also the line of code is much longer now with a lot more happening, which makes it visually heavier to read in the future.

Since this repeats, I'll mark every place that has exact same issue with: '<1>' comment.

Reply:
Resolved